tests/test_postfix.py

```diff
@@ -0,0 +1,34 @@
+"""This module contains the tests for the postfix container, the image with postfix, sendmail and mailq pre-installed."""
+
+import time
+
+from bci_tester.data import POSTFIX_CONTAINERS
+
+CONTAINER_IMAGES = POSTFIX_CONTAINERS
+
+
+def test_postfix_status(auto_container):
+    # sleep to allow completition of postfix configuration & service to start
+    time.sleep(20)
```

Comment:
Can this not be handled via a HEALTHCHECK or periodically trying another command? Arbitrary timeouts tend to flake often on slow CI workers.

Reply:
Added [HEALTHCHECK to the postfix containerfile](https://github.com/SUSE/BCI-dockerfile-generator/pull/1046/files#diff-70cc5e2f619e797806a06b44c8bc81cad4088a22eeebc3caaf7ddaa0885e042fR331-R332) -  (checking for "postfix status" to return 0).

Now with `postfix status` command added to the HEALTHCHECK itself, should I remove `test_postfix_status(...)` from here?

Reply:
Tested with HEALTHCHECK defined.
It's working (waiting for container to become healthy before executing tests.)


```
> OS_VERSION=tumbleweed TARGET=custom BASEURL=registry.opensuse.org/home/defolos/bci/staging/tumbleweed/tumbleweed-1046/ tox -e postfix

DEBUG    pytest_container:container.py:1147 Started container with 9938ade7ffba16cb925fd896697caa08840a455f361367b19ef5bf94d7f5d4d0 at 2024-05-30 11:55:05.554245
DEBUG    pytest_container:container.py:1159 Container has a healthcheck defined, will wait at most 55.0 s
DEBUG    pytest_container:container.py:1172 Container has the health status ContainerHealth.STARTING
DEBUG    pytest_container:container.py:1172 Container has the health status ContainerHealth.STARTING
DEBUG    pytest_container:container.py:1172 Container has the health status ContainerHealth.STARTING
DEBUG    pytest_container:container.py:1172 Container has the health status ContainerHealth.STARTING
DEBUG    pytest_container:container.py:1172 Container has the health status ContainerHealth.HEALTHY
```